frontend/src/app/modules/work_packages/routing/wp-list-view/wp-list-view.component.ts

```diff
@@ -54,7 +55,8 @@ import {QueryResource} from "core-app/modules/hal/resources/query-resource";
   providers: [
     { provide: HalResourceNotificationService, useClass: WorkPackageNotificationService },
     DragAndDropService,
-    CausedUpdatesService
+    CausedUpdatesService,
+    IfcModelsDataService
```

Comment:
Why is there a need to provide the IFC service into the base component which has nothing to do with IFC? Couldn't you simply inject it to the BCFListComponent, where it is actually needed?